pages/common/rdfind.md

```diff
@@ -0,0 +1,19 @@
+# rdfind
+
+> Finds duplicate files and gets rid of them.
+
+- Identify all duplicates in a given directory and outputs a summary:
+
+`rdfind -dryrun true {{path/to/directory}}`
+
+- Replace all duplicates with hardlinks:
+
+`rdfind -makehardlinks true {{path/to/directory}}`
+
+- Replace all duplicates with symlinks/soft links:
+
+`rdfind -makesymlinks true {{path/to/directory}}`
+
+- Delete all duplicates and do not ignore empty files:
+
+`rdfind -deleteduplicates true -ignoreempty false {{/mnt/backup}}`
```

Comment:
Why is this suddenly `/mnt/backup` ?

Reply:
Ooh good spot. I guess `path/to/backup` would have been better?

Reply:
or just set it to a generic pathname like the rest;